spec/envelope.md

```diff
@@ -0,0 +1,14 @@
+# Envelope
```

Comment:
Could this also link to statement.md?  that might help people follow along.

Reply:
this is now obsolete